content/blog/2024-06-20-etc-proof-of-work-course-31-pows-focus-on-trust-minimization/index.md

```diff
@@ -0,0 +1,103 @@
+
+We will explain the meaning of trust minimization and its benefits, and why it is the root of security in blockchains such as ETC and Bitcoin.
+
+Trust minimization is important because trusted third parties in traditional systems manage our information and wealth, but they can behave badly or be hacked by crooks. 
+
+Also, it is a general assumption in these systems that wherever trust is deposited, that trust will always be abused.
+
+## Satoshi Was Focused on Trust Minimization
+
+![](./1.png)
+
+POW is about trust minimization. 
+
+Proof of this is that Satoshi Nakamoto actually mentioned trust minimization [14 times](https://etherplan.com/2020/02/29/satoshi-nakamoto-mentioned-trust-minimization-14-times-in-the-bitcoin-white-paper/10210/) in the Bitcoin white paper. 
+
+For example, right from the get-go, in the abstract he wrote “but the main benefits are lost if a trusted third party is still required”.
+
+In the context of the blockchain industry, the term “security” means to depend the least possible on trusted third parties when it comes to money and applications to manage that money.
+
+Mr. Nakamoto was sharp-focused on trust minimization and the device he invented to achieve the decentralization needed to accomplish it was proof of work.
+
+## “Trustless” Is Not the Correct Term
+
+As my mentor, Nick Szabo, taught me, it is important to understand that POW is a trust minimization technology, not a “trustless” technology as frequently simplified. 
+
+Even in Bitcoin there is some minimal marginal dependency, thus trust, in the aggregate of miners, node operators, and core developers of the system.
+
+Indeed, if all node operators, miners, mining pools, and developers of Bitcoin were to agree on a nefarious change on the system, they could theoretically do it.
+
+However, this is highly unlikely because of what is called the “[coordination problem](https://ethereumclassic.org/blog/2023-03-02-ethereum-classic-course-11-ethereum-classic-social-and-physical-layer-security#4-social-layer-coordination-problem-security)”, which is a barrier to implementing unwanted changes due to Bitcoin’s physical decentralization and ability of participants to [split from the system](https://ethereumclassic.org/blog/2024-03-07-etc-proof-of-work-course-17-pow-has-division-of-power-pos-does-not#the-right-to-split) whenever they wish.
+
+## The Issue of Toxicity
+
+The difference between trustlessness and trust minimization is important to understand because it sheds additional light on the issue of “toxicity” and the need for strong opposition to bad ideas proposed for future upgrades of POW blockchains.
+
+Because these systems are truly decentralized, then there are no administrators or leaders dictating the direction of the projects.
+
+The only way to argue for or against a change is by sheer brute force on discussion channels and social media.
```

Comment:
?!?? does that mean twitter or other social media censors can prevent, or force, a change?